### PR TITLE
fix(helm): Chart meta

### PR DESCRIPTION
**Pull request by rarkins**

### CODE REVIEW

File: docs/installation-helm.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ Renovate On-Premises (v5 and earlier) was built with a "full" image only, so if
...
 
 ### Install Renovate chart
 
 ```shell
-helm install --generate-name --set renovate.config='\{\"token\":\"...\"\}' renovate-cc-ee/mend-renovate
+helm install --generate-name --set renovate.config='\{\"token\":\"...\"\}' mend-renovate-cc-ee/mend-renovate-ce
 ```
 
-See the available [values](../helm-charts/mend-renovate/values.yaml) for full configuration and review configuration guides for [GitHub](./configuration-github.md) and/or [GitLab](./configuration-gitlab.md).
+See the available [values](../helm-charts/mend-renovate-ce/values.yaml) for full configuration and review configuration guides for [GitHub](./configuration-github.md) and/or [GitLab](./configuration-gitlab.md).
```
Comment (rarkins) [inline docs/installation-helm.md:31]: These docs links also need updating

**Review by Gabriel-Ladzaretti — DISMISSED** [review]

File: docs/installation-helm.md (hunk 1 above)
Comment (Gabriel-Ladzaretti) [reply docs/installation-helm.md:31]: same line, `[GitHub](./configuration-github.md) and/or [GitLab](./configuration-gitlab.md).` is broken as well

Comment (Gabriel-Ladzaretti) [inline docs/installation-helm.md:31]:
```suggestion
See the available [values](../helm-charts/mend-renovate-ce/values.yaml) for full configuration and review configuration guides for [GitHub](./configure-renovate-ce-github.md) and/or [GitLab](./configure-renovate-ce-gitlab.md).

```

File: docs/installation-helm.md [hunk 2]
```diff
@@ -18,14 +18,14 @@ Renovate On-Premises (v5 and earlier) was built with a "full" image only, so if
...
-helm repo add renovate-ce https://mend.github.io/renovate-ce-ee
+helm repo add mend-renovate-ce https://mend.github.io/renovate-ce-ee
 helm repo update
 ```
 
 ### Install Renovate chart
 
 ```shell
-helm install --generate-name --set renovate.config='\{\"token\":\"...\"\}' renovate-cc-ee/mend-renovate
+helm install --generate-name --set renovate.config='\{\"token\":\"...\"\}' mend-renovate-cc-ee/mend-renovate-ce
```
Comment (Gabriel-Ladzaretti) [inline docs/installation-helm.md:28]:
```suggestion
helm install --generate-name --set renovate.config='\{\"token\":\"...\"\}' mend-renovate-ce/mend-renovate-ce
```
typo + its either we change the helm repo name here or when it is added L21.

File: docs/installation-helm.md [hunk 3]
```diff
@@ -18,14 +18,14 @@ Renovate On-Premises (v5 and earlier) was built with a "full" image only, so if
 ### Add Helm repository
 
 ```shell
-helm repo add renovate-ce https://mend.github.io/renovate-ce-ee
+helm repo add mend-renovate-ce https://mend.github.io/renovate-ce-ee
```
Comment (rarkins) [inline docs/installation-helm.md:21]:
```suggestion
helm repo add mend-renovate-ce-ee https://mend.github.io/renovate-ce-ee
```

**Review by Gabriel-Ladzaretti — APPROVED** [review]

**Pull request merged**